construct.yaml

```diff
@@ -41,6 +41,8 @@ specs:
   - cmreshandler >1.0.0b4
   - elasticsearch <7.14
   - elasticsearch-dsl
+  - opensearchpy
+  - opensearch-dsl
```

Comment:
What is the migration plan here?
* Remove `elasticsearch*` once we've had a few patch releases of `7.3.x` that support the OpenSearch client?
* Remove `elasticsearch*` when `8.0`+ is the only supported release?
* Depend on both forever?

Reply:
I am not entirely sure at what stage elasticsearch will be completely removed, but I believe we won't depend on both forever. @fstagni maybe can answer more precisely?

Reply:
We won't depend on both forever, but as long as we'll support DIRAC 7.3 we'll need to keep both of them.

Reply:
In that case, @rupozzi can you add a comment above the `elasticsearch` dependencies stating they can be removed once nobody is using DIRAC 7.3 any more?